app/models/sso_token.rb

```diff
@@ -3,6 +3,7 @@ class SSOToken
   include ActiveModel::MassAssignmentSecurity
   include ActiveModel::Serializers::Xml
   include Rails.application.routes.url_helpers
+  include DeveloperPortal::Engine.routes.url_helpers
```

Comment:
If we include both, I would prefer not including  any and use the `class_attribute` method

Because both can define the same routes names